packages/ts-transform/src/utils/css-transform.tsx

```diff
@@ -49,9 +49,9 @@ const parentOrphenedPseudos = plugin('parent-orphened-pseudos', () => {
 
 const extractStyleSheets = plugin<{ callback: (sheet: string) => void }>(
   'extract-style-sheets',
-  opts => {
-    return root => {
-      root.each(node => {
+  (opts) => {
+    return (root) => {
+      root.each((node) => {
```

Comment:
done with the shorthand expressions i see 😝

Reply:
oh i think prettier did it